File: src/native/lazero_kali_amd64/cognitionService/virtual/cog_core.erl
%S="hello world".
%Word="world".
%Ok = string:str(S, Word) > 0.
main2() ->
	S="hello world",
	Word="nope",
	Ok = string:str(S, Word) > 0,
	io:format(Ok).

main(String) ->
	main2(),
	S="hello world",
	Word="world",
	Ok = string:str(S, Word) > 0,
	io:format(Ok).
	%io:format("hello world").

File: src/native/lazero_kali_amd64/cognitionService/virtual/cog_core.php
<?php
//print("hello world")
$str = 'This is Main String';
 
if (strpos($str, 'This') !== false) {
    echo 'true';
}else{
	echo 'false';
}

//$obj= new ReflectionObject("str");
//$obj= new ReflectionClass($str).getMethods();
print_r($GLOBALS);
?>
